docs/make.bat

```diff
@@ -0,0 +1,36 @@
+@ECHO OFF
```

Comment:
Does this file really need to be checked in with the project?

Reply:
If we want to support Windows at some point I think it is not wrong to have this file in.

Reply:
Ok. I'll take a look at adding Windows into the Azure pipeline config file.

Reply:
https://github.com/tomtom-international/azure-pipeline-templates/pull/7

Reply:
Needed to fix this module on Windows and use ldap3 which is not depending on the ldap system dependency